### ISSUE TITLE
getHTML() Method in Toast-UI/Editor Returns Unexpected Value When Editor is Empty

### ISSUE THREAD

**Issue by susheelahegde** [nhn/tui.editor#3236]
## Describe the bug
When using the getHTML() method in toast-ui/editor, it consistently returns <p></br></p> instead of null when the editor is empty.

## To Reproduce

1. Initialize toast-ui/editor.
2. Use the getHTML() method when the editor is empty.
3. See that **"<p></br></p>"** is returned instead of **null**.

## Expected behavior
The **getHTML**() method should return null when the editor is empty just like **getMarkdown**().

## Screenshots
<img width="673" alt="image" src="https://github.com/nhn/tui.editor/assets/23210779/83574805-1c6f-4e1f-8564-d1ee24606836">

## Additional context
This bug affects the behavior of the **getHTML**() method specifically when the editor is empty. It may impact applications relying on the returned value to determine whether the editor content is empty or not.

**Comment by bacloud22** [comment]
I agree. Same for `getMarkdown` It grabs things from the HTML of the editor itself. Here in French language you see some values:

For me empty or not !

![image](https://github.com/nhn/tui.editor/assets/130276583/4bdbb202-09a4-4bd5-afbb-cee1b72f1036)
